proxy/grpcproxy/watch_broadcast.go

```diff
@@ -50,27 +50,24 @@ func newWatchBroadcast(wp *watchProxy, w *watcher, update func(*watchBroadcast))
 	wb.add(w)
 	go func() {
 		defer close(wb.donec)
-		// loop because leader loss will close channel
```

Comment:
watches that don't need requireleader will always fail even if they're opened at historical revisions